iron-overlay-behavior.html

```diff
@@ -477,7 +478,6 @@
           this._focusNode.focus();
         }
       } else {
-        this._focusNode.blur();
```

Comment:
Uhm, just tested this http://jsbin.com/vapoci/1/edit?html,console,output and it has different behavior on FF compared to Chrome, aka even if `display: none` the previously focused element is still the `document.activeElement`...Will put back this line where it was!